Use tool enabling static users generation for Dex (#1245)
kyma-project/kyma: resources/core/charts/cluster-users/README.md

## Overview

> The `cluster-users` sub-chart uses role-based access control (RBAC) to define roles and accesses in Kyma. Kyma does not have a user base, but it has an admin defined as **admin@kyma.cx**. The admin's password is generated and can be obtained from the secret named `admin-user` in the namespace of Dex release. Additionally, admin roles are bound to group of users named **kyma-admins**.

Reviewer: ```suggestion
The `cluster-users` sub-chart uses role-based access control (RBAC) to define roles and accesses in Kyma. Kyma does not have a user base, but it has an admin defined as **admin@kyma.cx**. The admin password is generated by Helm.  Get it from the `admin-user` Secret located in the Namespace of Dex release. Additionally, admin roles are bound to group of users named **kyma-admins**.
```
